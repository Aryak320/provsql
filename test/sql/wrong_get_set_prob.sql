\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

CREATE TABLE wrong_get AS SELECT get_prob(provenance()) FROM (SELECT DISTINCT city FROM personnel) t;
SELECT remove_provenance('wrong_get');
SELECT * FROM wrong_get;
SELECT set_prob(provenance(), 1.) FROM (SELECT DISTINCT city FROM personnel) t;
