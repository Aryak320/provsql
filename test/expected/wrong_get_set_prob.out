\set ECHO none
remove_provenance

(1 row)
get_prob



(3 rows)
ERROR:  set_prob called on non-input gate
